File: invesdwin-scripting-parent/invesdwin-scripting-julia/invesdwin-scripting-julia-runtime-contract/src/test/java/de/invesdwin/context/julia/runtime/contract/InputsAndResultsTestCharacter.jl
println("getCharacter")
if isdefined(Main, :getCharacter) && !isnothing(getCharacter)
	error("getCharacter already defined!")
end
getCharacter = putCharacter
println(typeof(getCharacter))
println(getCharacter)
if typeof(getCharacter) != Char
	error("getCharacter not Char!")
end

println("getCharacterVector")
if isdefined(Main, :getCharacterVector) && !isnothing(getCharacterVector)
	error("getCharacterVector already defined!")
end
getCharacterVector = putCharacterVector
println(eltype(getCharacterVector))
println(getCharacterVector)
if eltype(getCharacterVector) != Char
	error("getCharacterVector not Char!")
end

println("getCharacterVectorAsList")
if isdefined(Main, :getCharacterVectorAsList) && !isnothing(getCharacterVectorAsList)
	error("getCharacterVectorAsList already defined!")
end
getCharacterVectorAsList = putCharacterVectorAsList
println(eltype(getCharacterVectorAsList))
println(getCharacterVectorAsList)
if eltype(getCharacterVectorAsList) != Char
	error("getCharacterVectorAsList not Char!")
end

println("getCharacterMatrix")
if isdefined(Main, :getCharacterMatrix) && !isnothing(getCharacterMatrix)
	error("getCharacterMatrix already defined!")
end
getCharacterMatrix = putCharacterMatrix
println(eltype(getCharacterMatrix))
println(getCharacterMatrix)
if eltype(getCharacterMatrix) != Char
	error("getCharacterMatrix not Char!")
end

println("getCharacterMatrixAsList")
if isdefined(Main, :getCharacterMatrixAsList) && !isnothing(getCharacterMatrixAsList)
	error("getCharacterMatrixAsList already defined!")
end
getCharacterMatrixAsList = putCharacterMatrixAsList
println(eltype(getCharacterMatrixAsList))
println(getCharacterMatrixAsList)
if eltype(getCharacterMatrixAsList) != Char
	error("getCharacterMatrixAsList not Char!")
end

File: invesdwin-scripting-parent/invesdwin-scripting-runtime-lua/src/test/java/de/invesdwin/scripting/runtime/lua/tests/InputsAndResultsTestCharacter.lua
arraysToString = java.method(java.import('de.invesdwin.util.collections.Arrays'),'toString','java.lang.Object')

print("getCharacter")
if(getCharacter ~= nil) then
	error("getCharacter already defined!")
end
getCharacter = putCharacter
getCharacterType = type(getCharacter)
print(getCharacterType)
print(getCharacter)
if(getCharacterType ~= "string") then
	error("getCharacter not Character!")
end

print("getCharacterVector")
if(getCharacterVector ~= nil) then
	error("getCharacterVector already defined!")
end
getCharacterVector = putCharacterVector
getCharacterVectorType = type(getCharacterVector[1])
print(getCharacterVectorType)
print(arraysToString(getCharacterVector))
if(getCharacterVectorType ~= "string") then
	error("getCharacterVector not Character!")
end

print("getCharacterVectorAsList")
if(getCharacterVectorAsList ~= nil) then
	error("getCharacterVectorAsList already defined!")
end
getCharacterVectorAsList = putCharacterVectorAsList
getCharacterVectorAsListType = type(getCharacterVectorAsList[1])
print(getCharacterVectorAsListType)
print(arraysToString(getCharacterVectorAsList))
if(getCharacterVectorAsListType ~= "string") then
	error("getCharacterVectorAsList not Character!")
end

print("getCharacterMatrix")
if(getCharacterMatrix ~= nil) then
	error("getCharacterMatrix already defined!")
end
getCharacterMatrix = putCharacterMatrix
getCharacterMatrixType = type(getCharacterMatrix[1][1])
print(getCharacterMatrixType)
print(arraysToString(getCharacterMatrix))
if(getCharacterMatrixType ~= "string") then
	error("getCharacterMatrix not Character!")
end

print("getCharacterMatrixAsList")
if(getCharacterMatrixAsList ~= nil) then
	error("getCharacterMatrixAsList already defined!")
end
getCharacterMatrixAsList = putCharacterMatrixAsList
getCharacterMatrixAsListType = type(getCharacterMatrixAsList[1][1])
print(getCharacterMatrixAsListType)
print(arraysToString(getCharacterMatrixAsList))
if(getCharacterMatrixAsListType ~= "string") then
	error("getCharacterMatrixAsList not Character!")
end